cmd/build/helmify/replacements.go

```diff
@@ -114,6 +114,51 @@ var replacements = map[string]string{
 
 	"- HELMSUBST_MUTATION_STATUS_ENABLED_ARG": `{{ if not .Values.disableMutation}}- --operation=mutation-status{{- end }}`,
 
+	"- HELMSUBST_CONTROLLER_MANAGER_OPERATIONS": `
+        {{- if not .Values.controllerManager.disableWebhookOperation }}
+        - --operation=webhook
+        {{- end }}
+        {{- if not .Values.controllerManager.disableGenerateOperation }}
+        - --operation=generate
+        {{- end }}
+        {{- if not .Values.controllerManager.disableAuditOperation }}
+        - --operation=audit
+        {{- end }}
+        {{- if not .Values.controllerManager.disableStatusOperation }}
+        - --operation=status
+        {{- end }}
+        {{- if not .Values.controllerManager.disableMutationStatusOperation }}
+        - --operation=mutation-status
```

Comment:
`mutation-status` was meant to run from singleton pod to avoid write conflicts on mutation status. Lets remove this variable to make sure it cannot be set on controller-manager even by mistake. And if some user is asking for it specifically then we can add it later.